corehq/apps/analytics/static/analytix/js/hubspot.get_demo.js

```diff
@@ -0,0 +1,41 @@
+/* global window, hbspt */
+$(function () {
```

Comment:
This file should be requirejs compatible - I think we agreed on this when chatting offline, just leaving a note so I don't forget.

Reply:
correct. it will be resolved by https://github.com/dimagi/commcare-hq/pull/22503